chain/x/zoracle/internal/types/msgs.go

```diff
@@ -100,21 +100,24 @@ func (msg MsgRequestData) GetSignBytes() []byte {
 
 // MsgReportData is a message sent by each of the block validators to respond to a data request.
 type MsgReportData struct {
-	RequestID int64           `json:"requestID"`
-	DataSet   []RawDataReport `json:"dataSet"`
-	Sender    sdk.ValAddress  `json:"sender"`
+	RequestID      int64           `json:"requestID"`
+	RefundGasPrice int64           `json:"refundGasPrice"`
```

Comment:
Why we don't use sdk.Coins to represent GasPrice?

Reply:
I change RefundGasPrice to DecCoins because the gasPrice is also DecCoins